rand_isaac/Cargo.toml

```diff
@@ -18,9 +18,14 @@ travis-ci = { repository = "rust-lang-nursery/rand" }
 appveyor = { repository = "alexcrichton/rand" }
 
 [features]
-serde1 = ["serde", "serde_derive"] # enables serde for BlockRng wrapper
+serde1 = ["serde", "serde_derive", "rand_core/serde1"]
```

Comment:
Feature `serde1` implicitly implies `std` — should probably be documented with a comment here and in the README

Reply:
> Feature serde1 implicitly implies std

What do you mean? We don't require `std` for the `serde1` feature, and Serde works without `std`. We only need `std` for the tests.

Reply:
Oh, sorry. I mis-understood the double-negative conditional `no_std` attribute.